app/jobs/characterize_job.rb

```diff
@@ -2,9 +2,10 @@ class CharacterizeJob < ActiveJob::Base
   queue_as CurationConcerns.config.ingest_queue_name
 
   # @param [FileSet] file_set
-  # @param [String] filename a local path for the file to characterize. By using this, we don't have to pull a copy out of fedora.
+  # @param [String] filename a local path for the file to characterize so we don't have to pull a copy out of fedora.
   def perform(file_set, filename)
-    Hydra::Works::CharacterizationService.run(file_set, filename)
+    return unless file_set.characterization_proxy.content
```

Comment:
@mjgiarlo yeah, I see that. Although, thinking about it more, I can see two cases in which we might want to catch this.

1. An implementation that uses a `characterization_proxy` other than `:original_file`

It'll be up to the user to make sure whatever relation their using gets ingested. They'd do this by passing it to `IngestFileJob`:

https://github.com/projecthydra-labs/curation_concerns/blob/master/app/jobs/ingest_file_job.rb#L9

But if they haven't set the fileset's characterization proxy to the same relation, then things would fail.

2. `IngestFileJob` fails for some reason

I'm not seeing anything in `IngestFileJob` that would stop it from calling the characterization job if something went awry. It's possible that `CreateDerivativesJob` would succeed, though. However, I can see putting an exception here as helpful to the user, and taking the "fail faster" approach instead of letting the job continue when we know at least one part of it will fail.